bintray/conan_center_guide.rst

```diff
@@ -0,0 +1,88 @@
+.. _conan_center_flow:
+
+Contributing packages to conan-center
+=====================================
+
+As a moderated and curated repository, `conan-center`_ will not be populated automatically. Initially, it will be empty.
+To have your recipe or binary package available on `conan-center`_, you need to submit an inclusion request to Bintray,
+and the Bintray team will review your request.
+
+- If you are the author of an open source library, your package will be approved.
+  Keep in mind that it is your responsibility to maintain acceptable standards of quality for all packages you submit
+  for inclusion in `conan-center`_.
+
+- If you are packaging a third-party library you need to follow the guidelines below.
+
+
+Inclusion guidelines for third party libraries
+**********************************************
+
+In the inclusion request process, the JFrog staff will perform a general review and will make suggestions for improvements or
+better/cleaner ways to do implement the code.
+
+
+Recipe quality
+--------------
+
+- **Github public repository**: The recipe needs to be hosted in a public Github repository.
+
+- **Recipe fields**: `description`, `license` and `url` are required.
+
+- **Linter:** Is important to have a reasonable clean Linter, ``conan export`` and  ``conan test_package`` will
+  output some warnings and errors, keep it as clean as possible to guarantee a recipe less error prone and more understandable.
+
+- **Updated:** Not using deprecated features and when possible, using latest conan features, build helpers etc.
+
+- **Clean:** The code style will be reviewed to guarantee the readability of the recipe.
+
+- **test_package:** The recipes must contain a :ref:`test_package<packaging_getting_started>`
+
+
```

Comment:
- **License**: The package have to contain a LICENSE file with the library license.